portfolio/src/main/webapp/script.js

```diff
@@ -58,11 +58,13 @@ function getRandomName() {
 }
 
 /**
- * Fetches comments from DataServlet and adds them to the DOM as a list.
+ * Fetches the last commentsLimit comments from DataServlet and adds them to the DOM as a list.
+ * commentsLimit is selected by the user and sent to the server as parameter in the query string.
  */
 function getComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  const dataURL = `data?commentsLimit=${document.getElementById('commentsLimit').value}`;
 
+  fetch(dataURL).then(response => response.json()).then((comments) => {
```

Comment:
A good practice is to add TODOs for the future functionality that you might want to implement at some point in the future. It can be a good idea to add TODOs even if you're not planning to address them anytime soon. This is less of a commitment but an acknowledgement.

e.g:
// TODO(ak47na): Add support for error handling.

This makes it clear your intention to reviewer and marks potential extension points for future readers.